traffic_control/models/__init__.py

```diff
@@ -0,0 +1,7 @@
+from .barrier import *
```

Comment:
Import only the model classes instead of using `*` imports.

Reply:
Good point. Case handled.